day_7/10_little_monkeys.js

```diff
@@ -0,0 +1,70 @@
+/*
+## 10 Little Monkeys
+
+Create a file named `10_little_monkeys.js` and within that file, write a program that will print the following nursery rhyme, but for *10* monkeys.
+
+> Three little monkeys jumping on the bed,
+> One fell off and bumped his head,
+> Mama called the doctor and the doctor said,
+> "No more monkeys jumping on the bed!"
+>
+> Two little monkeys jumping on the bed,
+> One fell off and bumped his head,
+> Mama called the doctor and the doctor said,
+> "No more monkeys jumping on the bed!"
+>
+> One little monkey jumping on the bed,
+> He fell off and bumped his head,
+> Mama called the doctor and the doctor said,
+> "Get those monkeys right to bed!"
+
+### Bonus
+Can you write the program so that it will run for any number of monkeys?
+*/
+
+// Two different ways
+
+var firstRhyme = " little monkeys jumping on the bed, One fell off and bumped his head, Mama called the doctor and the doctor said, 'No more monkeys jumping on the bed!'"
+var secondRhyme = ' little monkey jumping on the bed, He fell off and bumped his head, Mama called the doctor and the doctor said, "Get those monkeys right to bed!"'
+
+for (var i = 10; i > 0; i--){//i equals 10 and i counts down stopping at 1
+  if (i > 1/*if i is greater than 1*/) console.log(i + /*log i + the following string to the console*/firstRhyme);
+  else if (i = 1/*if i equals 1*/) console.log(i + /*log i + the following string to the console*/secondRhyme);
+}
+
+for (var i = 10; i > 0; i--){//i equals 10 and i counts down stopping at 1
```

Comment:
How could you utilize an array with the word version of the numbers?

Reply:
You know it's funny but this problem kept bugging me and I didn't really feel right about it when I submitted it and then on June 4th (right when it was too late) I had that "a-ha" moment. This is what I came up with and it worked for me

`var firstRhyme = " little monkeys jumping on the bed, One fell off and bumped his head, Mama called the doctor and the doctor said, 'No more monkeys jumping on the bed!'"

var secondRhyme = ' little monkey jumping on the bed, He fell off and bumped his head, Mama called the doctor and the doctor said, "Get those monkeys right to bed!"'

var amountOfMonkeys = ["Ten", "Nine", "Eight", "Seven", "Six", "Five", "Four", "Three", "Two", "One"];

for (var i = 10; i > 0; i--){
  if (i > 1) console.log(amountOfMonkeys.shift() + firstRhyme);
  else if (i = 1) console.log(amountOfMonkeys.shift() + secondRhyme);
}`